### PR TITLE
updated

### PR DESCRIPTION
**Pull request by apachumbre**

### CODE REVIEW

File: server/bot/bot.js [hunk 1]
```diff
@@ -1759,6 +1760,8 @@ const sendProactiveMessageAsync = async (
     const activity = MessageFactory.attachment(
       CardFactory.adaptiveCard(approvalCard)
     );
+    //let titalmesg = `${incTitle} hiii`
+
```
Comment (dtiwarATS) [inline server/bot/bot.js:1764]: remove unwanted Code

File: server/bot/bot.js [hunk 2]
```diff
@@ -2019,6 +2022,24 @@ const sendProactiveMessageAsync = async (
               sendErrorEmail,
               retryCounter
             );
+            // sendProactiveMessaageToUserAsync(
```
Comment (dtiwarATS) [inline server/bot/bot.js:2025]: remove this code

**Review by dtiwarATS — CHANGES_REQUESTED** [review]
Please look at my comments

**Review by schowkekarats360 — APPROVED** [review]
Changes are looking good

**Pull request merged**